### ISSUE TITLE
[BUG] Keytips don't work as it was intended with non-english keyboard layout

### ISSUE THREAD

**Issue by Mant1kor** [obsproject/obs-studio#2772]
## Platform

Operating system and version: Windows 10
OBS Studio version: 25

## Expected Behavior

After OBS start, I press Alt and some key to use some feature.

## Current Behavior

After OBS start, I must every time change keyboard layout to English to use keytips.

## Steps to Reproduce

1. Use non-english OBS UI language.
2. Use non-english keyboard layout.
3. Try to use keytips.

## Additional information

Crowdin proofreaders are instructed: "Please keep the & character in your translation before the same character as in the source string. If this character is not present in your translation, put it at the end into brackets, like the other user did it."
And they "fixing" all other variants.

First - it looks ugly in UI.
Second - it breaks usability.

I suggest letting translators(if they know about collisions) to adapt keytips to their own languages.

**Comment by Vainock** [comment]
I also want to throw in a message I received from dodgepong after I asked him why translators are not allowed to change the letter after the &: (Why in each language the same alt keys need to be used.)
![_20200421_231136](https://user-images.githubusercontent.com/39059951/79963982-58e55800-848a-11ea-9e56-dbeb169acd0b.JPG)

I would like to support the idea from Mant1kor as this rule does not make any sense when the keyboard layout is different in another language, as these people just can't press the English alt keys.

The best would be just to give us as translators (and especially me as project proofreader who corrected each translation to match the instructions) the permission to change the letter after &.

**Comment by gxalpha** [comment]
@Vainock would you mind taking a look at this? Is this still a thing?

**Comment by Vainock** [comment]
@gxalpha Yes, this issue persists, and it is more a problem with the shortcuts in general than a bug in OBS or QT. Using a non-english keyboard layout does not work because you can't type latin characters. All shortcuts have been standardized on Crowdin, because I never understood why people wanted to change that (now I do!) and we were instructed by dodgepong to keep it consistent across all languages.
E.g. 'E&xit' is still 'خروج (&X)' in Arabic, though they don't even have access to that character on their keyboard:

Not really sure how we could approach that because the underlying problem can't be solved. What we could do is allowing translators to change the shortcuts how they want.

**Comment by Fenrirthviti** [comment]
Closing as stale. If this is still an issue please open with updated reproduction steps on the latest version of OBS.
